content/docs/learn/typed-errors/working-with-typed-errors.md

```diff
@@ -1,47 +1,102 @@
 ---
+description: Working, recovering, and accumulating errors in a typed and concise way.
 sidebar_position: 1
 ---
 
 # Working with typed errors
 
 <!--- TEST_NAME TypedErrorsTest -->
 
-Working with typed errors offers a few advantages over using exceptions:
+_Typed errors_ refer to a technique from functional programming in which we
+make _explicit_ in the signature (or _type_) the potential errors that may
+arise during the execution of a piece of code. This is not the case when using
+exceptions, which any documentation is not taken into account by the compiler,
+leading to a defensive mode of error handling.
 
-- **Type Safety:** Typed errors allow the compiler to find type mismatches early, making it easier to catch bugs before they make it to production. However, with exceptions, the type information is lost, making it more difficult to detect errors at compile-time.
+:::info Media resources
 
-- **Predictability:** When using typed errors, the possible error conditions are explicitly listed in the type signature of a function. This makes it easier to understand the possible error conditions and write tests covering all error scenarios.
+- [_Functional Error Handling - A Practical Approach_](https://kotlindevday.com/videos/functional-error-handling-a-practical-approach-bas-de-groot/) by Bas de Groot
+- [_Exception handling in Kotlin with Arrow_](https://www.youtube.com/watch?v=ipF540mBG9w) by Ramandeep Kaur
+- [_Por qué no uso excepciones en mi código_](https://www.youtube.com/watch?v=8WdprhzmQe4) by Raúl Raja and [Codely](https://codely.com/)
```

Comment:
@serras if you find it relevant, do you mind also putting my blog in this media links? https://medium.com/@mitchellyuwono/typed-error-handling-in-kotlin-11ff25882880

The reason why I believe this might help is many have reached out to me asking for real empirical evidence or code comparison as for "why" they should be using arrow for typed error handling.

Reply:
Done!